modules/blocks/process.js

```diff
@@ -165,33 +165,73 @@ Process.prototype.loadBlocksOffset = function (limit, offset, verify, cb) {
 				}
 
 				library.logger.debug('Processing block', block.id);
+
 				if (verify && block.id !== library.genesisblock.block.id) {
-					// Sanity check of the block, if values are coherent.
-					// No access to database.
-					var check = modules.blocks.verify.verifyBlock(block);
-
-					if (!check.verified) {
-						library.logger.error(['Block', block.id, 'verification failed'].join(' '), check.errors.join(', '));
-						// Return first error from checks
-						return setImmediate(cb, check.errors[0]);
-					}
-				}
-				if (block.id === library.genesisblock.block.id) {
-					modules.blocks.chain.applyGenesisBlock(block, cb);
+					async.series({
```

Comment:
Ok, don't care about that function - I scheduled it to be removed as part of https://github.com/LiskHQ/lisk/issues/544, because we don't benefit from it at all.